lib/reply.js

```diff
@@ -452,7 +463,10 @@ Reply.prototype.callNotFound = function () {
   return this
 }
 
+// TODO: should be removed in fastify@5
 Reply.prototype.getResponseTime = function () {
+  FSTDEP020()
```

Comment:
Can you add a test to check this is correctly emitted. You can modify one of the existing tests for `getResponseTime()`.